config/routes.rb

```diff
@@ -1,6 +1,14 @@
 Rails.application.routes.draw do
+  get 'likes/create'
+  get 'comments/new'
+  get 'comments/create'
+  get 'comment/new'
+  get 'comment/create'
   resources :users, only: [:index, :show] do
-    resources :posts, only: [:index, :show]
+    resources :posts do
```

Comment:
- [ ] `posts#create` routes is missing! Kindly ensure that there is a pair of routes in the _config/routes.rb_ file for creating posts, which the output is similar to:
```
  GET   /posts/new                            posts#new
  POST   /posts(.:format)                     posts#create
```

Reply:
Hello @rloterh 
Thank you for your feedback. The line you are referring to `line 3` is deleted(red). Could you please check the committed file and confirm.

Reply:
Hi @AdornChoga 
The comment was actually pointing to _line 8_, kindly add a corresponding _route_ for _create_ action in _posts controller_.

Reply:
Okay. Thank you for clarifying. :+1: